test/io/test_pcs_converter.py

```diff
@@ -339,7 +339,65 @@ def test_build_new_forbidden(self):
         cs.add_forbidden_clause(fb)
         value = pcs_new.write(cs)
         self.assertIn(expected, value)
-    
+
+    def test_build_new_GreaterThanFloatCondition(self):
+        expected = "b integer [0, 10] [5]\n" \
+                   "a real [0.0, 1.0] [0.5]\n\n" \
+                   "a | b > 5"
+        cs = ConfigurationSpace()
+        a = UniformFloatHyperparameter("a", 0, 1, 0.5)
+        b = UniformIntegerHyperparameter("b", 0, 10, 5)
+        cs.add_hyperparameter(a)
+        cs.add_hyperparameter(b)
+        cond = GreaterThanCondition(a, b, 5)
+        cs.add_condition(cond)
+
+        value = pcs_new.write(cs)
+        self.assertIn(expected, value)
```

Comment:
I'm wondering why you're not checking for equality here (and in all the other checks as well).